### PR TITLE
chore: publish prereleases to another repository

### LINKED ISSUE #8659: publish pre-releases to another repository
### Is your enhancement related to a problem? Please describe

we received over the past few years complaints that we've too many releases on this repository and it's hard to filter or to subscribe when a new release is out as from GitHub there is no way to easily filter out all pre-releases and keep only releases

### Describe the solution you'd like

publish our releases made after each commit to a separate repository

### Describe alternatives you've considered

_No response_

### Additional context

A suggestion was to also add in the details of the pre-release to add what's new in this pre-release (link to all commits / PR being merged) since the previous one

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
move post-commit/prereleases to https://github.com/containers/podman-desktop-prereleases repository
only main releases (like 1.12.0, 1.12.1, 1.13, etc will be released on podman-desktop repository)

### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop/issues/8659

### How to test this PR?

hard to manually test, need to like copy the workflow to your private fork, comment out lines using the token, use another repository where you have the rights

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

**Review by jeffmaury — APPROVED** [review]
LGTM

File: .github/workflows/next-build.yaml [hunk 1]
```diff
@@ -54,24 +57,31 @@ jobs:
...
           find extensions/* -maxdepth 2 -name "package.json" | xargs -I {} sed -i "s#version\":\ \"\(.*\)\",#version\":\ \"${{ steps.TAG_UTIL.outputs.desktopVersion }}\",#g" {}
+          # change the repository field to be the prerelease repository in package.json file
+          sed -i "s#\"repository\":\ \"\(.*\)\",#\"repository\":\ \"https://github.com/containers/podman-desktop-prereleases\",#g" package.json
           cat package.json
           git add package.json extensions/*/package.json
           git commit -m "chore: tag ${{ steps.TAG_UTIL.outputs.githubTag }}"
           echo "Tagging with ${{ steps.TAG_UTIL.outputs.githubTag }}"
           git tag ${{ steps.TAG_UTIL.outputs.githubTag }}
-          git push origin ${{ steps.TAG_UTIL.outputs.githubTag }}
+          # push tag to the preleases repository
```
Comment (feloy) [inline .github/workflows/next-build.yaml]:
```suggestion
          # push tag to the prereleases repository
```

**Review by feloy — APPROVED** [review]

**Review by axel7083 — APPROVED** [review]

**Comment by axel7083** [comment]
Should all the draft release in this repository be deleted ? or do we keep the previous ?

cc @slemeur @jeffmaury

**Comment by benoitf** [comment]
goal is to remove them (including tags)

**Pull request merged**